server/controllers/BookmarkController.js

```diff
@@ -80,6 +79,40 @@ class BookmarkController {
       })
```

Comment:
Please what is this create method doing here?

Reply:
The create method creates a row in the bookmark table.